### DISCUSSION TITLE
Font Not Displaying on PWA

### DISCUSSION THREAD

**Discussion by countrymusicfy** [AzuraCast/AzuraCast#4326] [Q&A]
Hi, I downloaded my PWA on Android but the 'Magneto' font from the public player on the website is not displaying on the PWA. I am guessing that I need to use @font-face and reference the 'Magneto' file source in the custom css. Does anyone know the file source for fonts on azuracast?

**Comment by BusterNeece** [comment]
I've created a GitHub issue about this, as this is something we need to add to our cache for the service worker. #4327
